sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/persons/PersonDirectorySteps.java

```diff
@@ -246,14 +246,14 @@ public PersonDirectorySteps(
           webDriverHelpers.clickOnWebElementBySelector(By.cssSelector("[role='gridcell'] a"));
         });
 
-      When(
-              "I apply on the APPLY FILTERS button",
-              () -> {
-                  webDriverHelpers.waitUntilIdentifiedElementIsVisibleAndClickable(
-                          APPLY_FILTERS_BUTTON, 30);
-                  webDriverHelpers.clickOnWebElementBySelector(APPLY_FILTERS_BUTTON);
-                  TimeUnit.SECONDS.sleep(10);
-              });
+    When(
+        "I apply on the APPLY FILTERS button",
```

Comment:
"I click on"
Add location of the action

Reply:
I just reused existing step, if change required I Will adjust naming